QSB/Tools/PlayerToolsManager.cs

```diff
@@ -124,7 +124,7 @@ private static void CreateTranslator()
             tool.ArrivalDegrees = 5f;
             tool.Type = ToolType.Translator;
             tool.ToolGameObject = group.gameObject;
-            oldTranslator.enabled = false;
+            Object.Destroy(oldTranslator);
```

Comment:
even if it's enabled, Awake is called, which requires the destroyed translator prop